File: Code/Tier3-DatabaseLayer/RestServer/Common/MeetingNotes.cs
using System.ComponentModel.DataAnnotations;

namespace WebApplication.Common
{
    public class MeetingNotes
    {
        public int groupId { get; set; }
        public string planningMeeting { get; set; }
        public string retrospectiveMeeting { get; set; }
        public string reviewMeeting { get; set; }
        public string dailyMeeting { get; set; }
        public int SprintId { get; set; }
        public bool validated { get; set; }
      

        public MeetingNotes()
        {
        }
        public MeetingNotes(string planningMeeting, string retrospectiveMeeting, string reviewMeeting, string dailyMeeting)
        {
            this.planningMeeting = planningMeeting;
            this.retrospectiveMeeting = retrospectiveMeeting;
            this.reviewMeeting = reviewMeeting;
            this.dailyMeeting = dailyMeeting;
        }

        
    }
}

File: Code/Tier2/Common/src/main/java/javaModel/MeetingNotes.java
package javaModel;

import java.io.Serializable;

public class MeetingNotes implements Serializable {

    private String planningMeeting;
    private String retrospectiveMeeting;
    private String reviewMeeting;
    private String dailyMeeting;
    private int sprintId;
    private boolean validated;
    private int groupId;

    public int getGroupId() {
        return groupId;
    }

    public void setGroupId(int groupId) {
        this.groupId = groupId;
    }

    public boolean isValidated() {
        return validated;
    }

    public void setValidated(boolean validated) {
        this.validated = validated;
    }

    public MeetingNotes() {

    }

    public int getSprintId() {
        return sprintId;
    }

    public void setSprintId(int sprintId) {
        this.sprintId = sprintId;
    }

    public MeetingNotes(String planningMeeting, String retrospectiveMeeting, String reviewMeeting, String dailyMeeting) {
        this.planningMeeting = planningMeeting;
        this.retrospectiveMeeting = retrospectiveMeeting;
        this.reviewMeeting = reviewMeeting;
        this.dailyMeeting = dailyMeeting;

    }

    public String getPlanningMeeting() {
        return planningMeeting;
    }

    public void setPlanningMeeting(String planningMeeting) {


        this.planningMeeting = planningMeeting;

    }

    public String getRetrospectiveMeeting() {
        return retrospectiveMeeting;
    }

    public void setRetrospectiveMeeting(String retrospectiveMeeting) {



        this.retrospectiveMeeting = retrospectiveMeeting;

    }

    public String getReviewMeeting() {
        return reviewMeeting;
    }


    public void setReviewMeeting(String reviewMeeting) {



        this.reviewMeeting = reviewMeeting;

    }

    public String getDailyMeeting() {
        return dailyMeeting;
    }

    public void setDailyMeeting(String dailyMeeting) {
       this.dailyMeeting = dailyMeeting;
    }

}
